src/App.jsx

```diff
@@ -1,33 +1,52 @@
-import { Outlet } from 'react-router-dom';
+import { Routes, Route, BrowserRouter } from 'react-router-dom';
 import Footer from './components/layout/footer/Footer';
 import Header from './components/layout/header/Header';
 import { useKeycloak } from '@react-keycloak/web';
 import { useEffect } from 'react';
 import { ApplicationProvider } from './context/ApplicationContext';
 import ErrorBoundary from './components/error/error-boundary/ErrorBoundary';
+import FormsExample from './pages/FormsExample';
+import Home from './pages/Home';
+import Timecard from './pages/timecard/Timecard';
+import Accruals from './pages/accruals/Accruals';
 
 const App = () => {
   const { initialized, keycloak } = useKeycloak();
 
-  useEffect(() => {
+  useEffect(async () => {
     if (initialized && !keycloak.authenticated) {
-      keycloak.login();
+      await keycloak.login();
     }
   });
 
   return keycloak.authenticated ? (
     <ApplicationProvider>
-      <div className="App">
-        <Header />
-        <div className="govuk-width-container">
-          <main className="govuk-main-wrapper" id="main-content" role="main">
-            <ErrorBoundary>
-              <Outlet />
-            </ErrorBoundary>
-          </main>
+      <BrowserRouter>
+        <div className="App">
+          <Header />
+          <div className="govuk-width-container">
+            <main className="govuk-main-wrapper" id="main-content" role="main">
+              <ErrorBoundary>
+                <Routes>
+                  <Route index="true" element={<Home />} />
+                  <Route path="/timecard/:date" element={<Timecard />} />
+                  <Route path="/accruals/:date" element={<Accruals />} />
+                  <Route path="/forms" element={<FormsExample />} />
```

Comment:
Could we remove this and the FormsExample component while we are amending the routing?

Reply:
Removed FormsExample from routing and code